src/UI/Components/Diagram/components/RightSidebar.tsx

```diff
@@ -174,6 +152,12 @@ export const RightSidebar: React.FC<Props> = ({ editable }) => {
       setAttributes(instanceAttributes);
     }
 
+    if (stencilName) {
```

Comment:
You can avoid the if condition by doing:
`setIsInterServiceRelation(!!stencilName);`